ios/FluentUI/Core/Theme/Tokens/AliasTokens.swift

```diff
@@ -423,6 +423,172 @@ public final class AliasTokens {
         case brandStroke1
         case brandStroke1Pressed
         case brandStroke1Selected
+        // Dark red
```

Comment:
A bit annoying, though it should be fast-ish, but could you put these in the same order as the `SharedColorSets` enum?

Reply:
Ah, actually, never mind. The actual global tokens aren't even in the same order as the enum.

Reply:
I think probably reorder to match the enum, but let's check with @mischreiber 

Somewhat relevant comment from the PR they added the enum in:
https://github.com/microsoft/fluentui-apple/pull/917#discussion_r810414366

Reply:
are we actually planning on using these shared colors? lets not arbirarily add them if we aren't going to use them.

Reply:
They've been defined for avatar colors.